frontend/src/pages/AllChatsPage.jsx

```diff
@@ -23,16 +23,50 @@ const ChatInboxHeader = () => (
     </div>
 )
 
+// find the blocked accounts of the user
+function useFetchBlockedUsers(myUid) {
```

Comment:
For a future PR: We could move this function to `app/data/blocked.js` and then import it here. That way the function can be re-used in places where block data is needed. This could be done in a future PR, that way it can be reused both here, and in the block-users feature.